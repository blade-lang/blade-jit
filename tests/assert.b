try {
  assert 5 == 5
  assert [] == [1], 'Non empty list expected'
} catch e {
  echo e.stacktrace
}
