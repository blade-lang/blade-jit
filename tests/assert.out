>>>
AssertError\: Non empty list expected.*assert\.b\:3\:1 \-\> @\.script\(\)